WB-355: make Strut truly incompressible

kevinbarabash opened the pull request:
Before:
<img width="958" alt="screen shot 2018-08-09 at 6 47 37 pm" src="https://user-images.githubusercontent.com/1044413/43929980-26a39d94-9c05-11e8-99d4-34f0b928d084.png">

After:
<img width="970" alt="screen shot 2018-08-09 at 6 46 51 pm" src="https://user-images.githubusercontent.com/1044413/43929970-1fa9c37e-9c05-11e8-9b73-f9b10b4ab483.png">

jeresig commented:
Deploy preview for *wonder-blocks* ready!

Built with commit 0e634413545278f0fd6d0fbe97666b26f094b005

https://deploy-preview-274--wonder-blocks.netlify.com

kevinbarabash commented:
@jeresig I added a `strut.md` and added an example within it.

<img width="966" alt="screen shot 2018-08-09 at 8 46 32 pm" src="https://user-images.githubusercontent.com/1044413/43933040-54f6e632-9c15-11e8-9830-e19a061473d2.png">
